src/components/chip.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { ButtonBase } from '../base-components'
+
+
+const Chip = ({ classes, children, startIcon, endIcon, color, disabled, ...rest }) => {
+  const chipClasses = Object.freeze({
+    button: `
+      ${classes.chip ? classes.chip : 'px-1.5 p-py rounded-md'}
+      border bg-${color}-100 border-${color}-100 fill-current text-${color}-500
```

Comment:
For string concatenations like `bg-${color}-100` tailwind won't compile these properly in production, i've tested with joining like: `['bg', color, '100].join('-')` and it works, suggest to change to join from an array, assign to a variable, then place it into the classes template literal

Reply:
I see. I am gonna address this problem in the PR #24 with all my others components together since they will need to be changed as well

Reply:
sounds good 👍